src/types.js

```diff
@@ -11,6 +11,13 @@
  *   outermost element.
  */
 
+/**
+ * Props common to components that are opinionated compositions of other
+ * components.
```

Comment:
Is this only for components which are "simple" combinations of other components? I expect we might in future have composite components which are somewhat more complex than say `ScrollBox`.

Reply:
Let's see how this pans out when we implement `Table`, which is coming up soon. That's probably on the more complex side of any _common_ component we'd have.